content/blog/2021-09-21-collaborative-experiments.md

```diff
@@ -0,0 +1,158 @@
+feedback._
+
+The way DVC works is by storing custom Git refs in your repo with metadata that
+defines the experiment. You can learn more about how DVC uses custom Git refs in
+[this post](https://dvc.org/blog/experiment-refs).
+
+Next, you'll need to set up a remote to your data location. This could be an AWS
+S3 bucket, a Google Drive, or
+[one of the other supported storage types](https://dvc.org/doc/command-reference/remote/add#supported-storage-types).
+For this example, we'll be using a Google Drive folder as the remote to handle
+data storage.
+
+You can learn more about the Google Drive setup in
+[the docs](https://dvc.org/doc/command-reference/remote/add). The main thing you
+need to know is that you need the folder id for the remote location you want to
+set up. You can find that id in the URL for that folder. It'll look similar to
+this.
+
+```dvc
+https://drive.google.com/drive/folders/1k6aUYWphOulJlXgq4XbfKExWGyymTpEl?usp=sharing
+```
+
+Now that you know what we're doing, let's run the command to set up our DVC
+remote.
+
+```dvc
+dvc remote add -d cloud_remote gdrive://1k6aUYWphOulJlXgq4XbfKExWGyymTpEl
+```
+
+This adds the remote storage named `cloud_remote` for DVC to track and we'll be
+able to push and pull the exact code and data to reproduce any experiment from
+end to end. With your Git remote and DVC remote in place, you can start pulling
+data and experiments from the cloud to your local machine.
+
+## Listing your remote experiments
+
+When you're working with a team on an existing project, you might want to see
+the experiments already in the remotes so you know what's available. You can
+take a look at all of the experiments with the following command.
+
+```dvc
+dvc exp list origin --all
+```
+
+You'll get a list of all of the experiments across different Git branches that
+have been pushed with DVC. The output will look similar to this.
+
+```dvc
+21784fa:
+        exp-c8dcf
+main:
+        exp-b3667
+        exp-d382a
+```
+
+Now you'll be able to pick which experiment you want to reproduce and start
+testing with.
+
+## Pulling experiments
+
+If you're picking up an existing project, there will likely be a specific
+experiment you'll get started with. To pull an experiment to your local machine,
+you'll need an experiment id for the following command.
+
+```dvc
+dvc exp pull origin exp-b3667
+```
+
+The `exp-b3667` comes from the `dvc exp list` command we ran earlier and now you
+have all of the data and code associated with that experiment on your machine.
+
+From here, you can start running new experiments with different models,
+hyperparameters, or even datasets.
+
+## Pushing experiments
+
+Once you're done with your new experiments, you can push these to the same
+remote you pulled from. DVC handles both the GitHub and data storage pushes with
+this command.
+
+```dvc
+dvc exp push origin exp-p4202
+```
+
+This will push the custom Git refs to your repo and it will push any artifacts,
```

Comment:
@flippedcoder @dberenbaum Could we add a note indicating that `dvc exp push` also push the `checkpoints` of an experiment (if any). 

It is one of the few ways that we currently support in order to properly share experiments with checkpoints.